addons/binding/org.openhab.binding.evohome/src/main/java/org/openhab/binding/evohome/EvohomeBindingConstants.java

```diff
@@ -27,22 +27,28 @@
     // List of all Thing Type UIDs
     public static final ThingTypeUID THING_TYPE_EVOHOME_GATEWAY        = new ThingTypeUID(BINDING_ID, "gateway");
     public static final ThingTypeUID THING_TYPE_EVOHOME_DISPLAY        = new ThingTypeUID(BINDING_ID, "display");
-    public static final ThingTypeUID THING_TYPE_EVOHOME_RADIATOR_VALVE = new ThingTypeUID(BINDING_ID, "radiatorValve");
     public static final ThingTypeUID THING_TYPE_EVOHOME_LOCATION       = new ThingTypeUID(BINDING_ID, "location");
+    public static final ThingTypeUID THING_TYPE_EVOHOME_HEATING_ZONE   = new ThingTypeUID(BINDING_ID, "heatingzone");
+    public static final ThingTypeUID THING_TYPE_EVOHOME_RADIATOR_VALVE = new ThingTypeUID(BINDING_ID, "radiatorValve");
```

Comment:
Yeah it can't - i've not tided up yet - as you had an alpha version out I wanted to try to get this in as soon as possible. Happy for you to change or I will do a clean up once code is nicer.